modules/server/src/main/ts/bedrock/compiler/Imports.ts

```diff
@@ -52,16 +63,20 @@ const addTest = (testFilePath: string) => {
     console.error('no test list to add tests to');
   }
 };
+const handleParseError = (testFilePath: string, error: any) => {
+  ${useRequire ? 'const UnitTest = require(\'@ephox/bedrock-client\').UnitTest;' : 'import { UnitTest } from \'@ephox/bedrock-client\';'}
```

Comment:
Given the amount of escaping failures possible in statements like these, could we load these snippets from a file? Could that make it easier?

Reply:
I guess the parameterisation of it probably makes it harder.

Reply:
Yeah, most of this is dynamic and hence can't be in a file. We could look at making a template file for this, but I'm not convinced it's worth it as it'd still run into similar issues.